backend/server.js

```diff
@@ -0,0 +1,247 @@
+var express = require('express'); 
```

Comment:
Codacy has a fix for the issue: [Replace `'express');·` with `"express");`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
var express = require("express");
```